### ISSUE TITLE
Making a commit to any branch runs the deployment

### ISSUE THREAD

**Issue by caprenter** [notshi/mainhobby#9]
Is this expected?

My workflow is generally - create an issue, create a branch, commit a change on that branch, create a pull request. Would then expect a build to happen once that is merged into the 'build' branch

**Comment by notshi** [comment]
Yes, as designed. The plated framework was created without branches in mind (& for small projects between trusted devs). It's a different kind of workflow - hope that's ok!

**Comment by caprenter** [comment]
Got it !

thanks!
